composer.json

```diff
@@ -27,7 +27,7 @@
         "symfony/assetic-bundle": "~2.3",
         "symfony/swiftmailer-bundle": "~2.3",
         "symfony/monolog-bundle": "~2.4",
-        "symfony-cmf/symfony-cmf": "1.2.*",
+        "symfony-cmf/symfony-cmf": "1.3.*",
```

Comment:
I guess this requires https://github.com/symfony-cmf/symfony-cmf/pull/222 to be merged ?